gcmr/assembly/transient/Griffin_transient.i

```diff
@@ -0,0 +1,220 @@
+    collapse_scattering  = true
+    hide_angular_flux = true
+  []
+[]
+
+[GlobalParams]
+  library_file = '../ISOXML/XS_Griffin.xml'
+  library_name = XS_Griffin
+  isotopes = 'pseudo'
+  densities = 1.0
+  is_meter = true
+# power normalization
+  plus = true
+  dbgmat = false
+  grid_names = 'Tmod'
+  grid_variables = 'Tf'
+[]
+
+[PowerDensity]
+  power = 225e3 # Assembly Power from NS
+  power_density_variable = power_density
+  integrated_power_postprocessor = integrated_power
+[]
+
+[Materials]
+  [mod]
+    type = CoupledFeedbackMatIDNeutronicsMaterial
+    block = '10 100 102 103 200 201 300 400 401 500 600 8000 8001'
+    # TODO: Add changes in densities
+  []
+[]
+
+[MultiApps]
+  [bison]
+    type = TransientMultiApp
+    positions = '0 0 0'
+    input_files  = BISON_tr.i
+    execute_on = 'initial timestep_end'
+    # no_backup_and_restore = true # to restart from the latest solve of the multiapp (for pseudo-transient)
+    # keep_solution_during_restore = true
+    #app_type = SamApp
+    #library_name = libsam-opt.la
+    #library_path = '/beegfs1/software/NEAMS_microreactor/projects_super_mar22/SAM/lib'
+  []
+[]
+
+[Transfers]
+  [to_sub_power_density]
+    type = MultiAppProjectionTransfer
+    direction = to_multiapp
+    multi_app = bison
+    variable = power_density
+    source_variable = power_density
+    execute_on = 'initial timestep_end'
+    displaced_source_mesh = false
+    displaced_target_mesh = false
+    use_displaced_mesh = false
+  []
+  [from_sub_temp]
+    type = MultiAppInterpolationTransfer
+    direction = from_multiapp
+    multi_app = bison
+    variable = Tf
+    source_variable = Tfuel
+    execute_on = 'initial timestep_end'
+    displaced_source_mesh = false
+    displaced_target_mesh = false
+    use_displaced_mesh = false
+    num_points = 1 # interpolate with one point (~closest point)
+    power = 0 # interpolate with constant function
+  []
+[]
+
+[Postprocessors]
+  [scaled_power_avg]
+    type = ElementAverageValue
+    block = 'Fuel Fuel_tri'
+    variable = power_density
+    execute_on = 'initial timestep_end'
+  []
+  [fuel_temp_avg]
+    type = ElementAverageValue
+    variable = Tf
+    block = 'Fuel Fuel_tri'
+    execute_on = 'initial timestep_end'
+  []
+  [fuel_temp_max]
+    type = ElementExtremeValue
+    value_type = max
+    variable = Tf
+    block = 'Fuel Fuel_tri'
+    execute_on = 'initial timestep_end'
+  []
+  [fuel_temp_min]
+    type = ElementExtremeValue
+    value_type = min
+    variable = Tf
+    block = 'Fuel Fuel_tri'
+    execute_on = 'initial timestep_end'
+  []
+[]
+
+[UserObjects]
+  [ss]
+    type = TransportSolutionVectorFile
+    transport_system = SN
+    writing = false
+    execute_on = initial
+  []
+[]
+
+
+[Outputs]
+  interval = 1
+#  [exodus]
+#    type = Exodus
+#  []
+  [csv]
+    type = CSV
+  []
+#  [cp]
+#    type = Checkpoint
+#    interval = 25
+#  []
+  checkpoint = true
+  color = true
+  perf_graph = true
+[]
```

Comment:
You will need to add `tests` file so that CIVET can test those inputs. If the model is too heavy, the test can be in type of `RunApp` with `--check-input` option.